CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing guide 
+
+Contributions and feedback on your experience of using this software are welcome.
+
+This includes bug reports, feature requests, ideas, pull requests, and examples of how you have used this software.
+
+Please see the [Code of Conduct](CODE_OF_CONDUCT.md) and follow any templates configured in GitHub when reporting bugs, requesting enhancements, or contributing code.
+
+Please raise any significant new functionality or breaking change an issue for discussion before raising a Pull Request for it.
+
+## For contributors
+
+Anyone can be a contributor. Either you found a typo, or you have an awesome feature request you could implement, we encourage you to create a Pull Request.
+
+### Pull Requests
+
+- The latest changes are always in `canary`, so please make your Pull Request against that branch.
+- Pull Requests should be raised for any change
+- Pull Requests need approval of a [core contributor](https://next-auth.js.org/contributors#core-team) before merging
+- Rebasing in Pull Requests is preferred to keep a clean commit history (see below)
+- Run `npm run lint:fix` before committing to make resolving conflicts easier (VSCode users, check out [this extension](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs) to fix lint issues in development)
+- We encourage you to test your changes, and if you have the opportunity, please make those tests part of the Pull Request
+- If you add new functionality, please provide the corresponding documentation as well and make it part of the Pull Request
+
+### Setting up local environment
+
+A quick guide on how to setup _next-auth_ locally to work on it and test out any changes:
+
+1. Clone the repo:
+
+```sh
+git clone git@github.com:nextauthjs/next-auth.git
+cd next-auth
+```
+
+2. Install packages:
+
+```sh
+npm i
+```
+
+3. Populate `.env.local`:
+
+   Copy `.env.local.example` to `.env.local`, and add your env variables for each provider you want to test.
+
+> NOTE: You can add any environment variables to .env.local that you would like to use in your dev app.
+> You can find the next-auth config under`pages/api/auth/[...nextauth].js`.
+
+1. Start the dev application/server and CSS watching:
+
+```sh
+npm run dev
+```
```

Comment:
I think the `dev` script as is today masks log messages from the terminal because of the CSS watcher, I myself started using `yarn next` to get around it... Adapter contributors will likely not change the CSS anyway, so I think it is OK to change the required script to run to `npx next`?

Reply:
I have clean up `CONTRIBUTING.md` as it was just copied